src/fusion_power_plant.cc

```diff
@@ -43,7 +44,12 @@ void FusionPowerPlant::EnterNotify() {
   fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
     (kDefaultTimeStepDur * 12) * context()->dt());
   blanket_turnover = blanket_size * blanket_turnover_fraction;
-  double startup_inventory = reserve_inventory + sequestered_equilibrium; 
+  startup_inventory = reserve_inventory + sequestered_equilibrium; 
```

Comment:
1. Does  `startup_inventory` need to be a member variable if it is only used here?
2. If we define it after `tritium_overbuy_multiplier` (or whatever we end up calling it), can't we just define `startup_inventory` to include that effect rather than multiplying in the initializer for the buy policy?  That is, isn't the actual startup inventory inclusive of the decay effect?

Reply:
See previous comment about removing this. I think it ended up feeling too kludgey...